os/fs/procfs/fs_procfsproc.c

```diff
@@ -407,27 +410,49 @@ static ssize_t proc_entry_stat(FAR struct proc_file_s *procfile, FAR struct tcb_
 #else
 	ppid = -1;
 #endif
-	linesize = snprintf(procfile->line, STATUS_LINELEN, "%d %d %d %d %d %d %d %d %d", tcb->pid, ppid, tcb->sched_priority, tcb->flags, tcb->task_state, tcb->adj_stack_size, peak_stack, curr_heap, peak_heap);
+	linesize = snprintf(procfile->line, STATUS_LINELEN, "%d %d %d %d %d %d %d %d %d ", tcb->pid, ppid, tcb->sched_priority, tcb->flags, tcb->task_state, tcb->adj_stack_size, peak_stack, curr_heap, peak_heap);
 	copysize = procfs_memcpy(procfile->line, linesize, buffer, buflen, &offset);
 	totalsize += copysize;
 
 #ifdef CONFIG_SCHED_CPULOAD
 	for (cpuload_idx = 0; cpuload_idx < SCHED_NCPULOAD; cpuload_idx++) {
-		buffer += copysize;
-		remaining -= copysize;
-		if (totalsize >= buflen) {
-			return totalsize;
-		}
-
 		/* Get cpuload measurement data */
 		(void)clock_cpuload(procfile->pid, cpuload_idx, &cpuload);
-		if (cpuload.total > 0) {
-			load_value = (double)(100 * cpuload.active) / (double)cpuload.total;
+
+		for (int i = 0; i < CONFIG_SMP_NCPUS; i++) {
+			buffer += copysize;
+			remaining -= copysize;
+			if (totalsize >= buflen) {
+				return totalsize;
+			}
+
+			if (cpuload.total[i] > 0) {
+				load_value = (double)(100 * cpuload.active[i]) / (double)cpuload.total[i];
+				total_cpuload += cpuload.total[i];
+				total_active += cpuload.active[i];
+			} else {
+				load_value = 0.0f;
+			}
+
+			linesize = snprintf(procfile->line, STATUS_LINELEN, "%.1f-", load_value);
+			copysize = procfs_memcpy(procfile->line, linesize, buffer, remaining, &offset);
+			totalsize += copysize;
+
+		}
+
+		if (total_cpuload > 0) {
```

Comment:
`total_cpuload` will be the total load of all the cpu in the system. However, its value needs to be between 0 - 100%. So, we need to take average of all the cpu loads here. 
`total_cpuload = total_cpuload / CONFIG_SMP_NCPUS;`

Please change accordingly at all required points.